libs/daedalus/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "daedalus"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+proc-macro = true
+
+[dependencies]
+cangen = { path = "../cangen" }
```

Comment:
try and use workspace cangen for this.  It may not work but if it does could optimize builds.

Reply:
I get an error when I try to do this:
```
error: multiple workspace roots found in the same workspace:
/home/harry/NER/Calypso/libs/daedalus
/home/harry/NER/Calypso
```
I did some research and this seems to be a puzzling subject matter. I'm not super well-versed in cargo directives so we can go over it more tomorrow if you think it's worth looking into.